Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java

```diff
@@ -82,11 +86,15 @@ private void setUpMotorTeam(XCANTalon master) {
         master.configPeakOutputVoltage(12, -6);
 
         master.setProfile(0);
-        master.setF(.099);
         updateMotorPidValues(master);
         master.setControlMode(TalonControlMode.Speed);
-
         master.set(0);
+        
+        follower.setProfile(0);
+        follower.setControlMode(TalonControlMode.Follower);
+        follower.configNominalOutputVoltage(0,  -0);
+        follower.configPeakOutputVoltage(-12, 6);
```

Comment:
I think the proper way to invert this is to call `follower.setInverted(true)`, though we should verify that actually works on-robot =]